src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -880,7 +880,12 @@ module Types = struct
             ~doc:"Amount of coda granted to the producer of this block"
             ~args:Arg.[]
             ~resolve:(fun _ {coinbase; _} -> Currency.Amount.to_uint64 coinbase)
-        ] )
+        ; field "coinbaseReceiverAccount"
+            ~typ:(non_null AccountObj.account)
```

Comment:
Can you confirm with @deepthiskumar that we patched the bug where blocks were created with coinbases sometimes?

Reply:
Ah do we return the "empty" account in that case? I think it would be better to make it an option instead.

Reply:
O.O it's possible to have a block with no coinbase? cc @deepthiskumar

Reply:
Yea, it is possible to have blocks without coinbase (when no snarks are available to purchase)

Reply:
Ok thanks, will make the receiver optional (we make the coinbase amount zero when there's no coinbase for reference)